common/lib/dependabot/update_checkers/version_filters.rb

```diff
@@ -0,0 +1,17 @@
+module Dependabot
+  module UpdateCheckers
+    module VersionFilters
+      def filter_vulnerable_versions(versions_array)
+        versions_array.reject do |v|
+          security_advisories.any? do |a|
+            if a.is_a?(Gem::Version)
```

Comment:
should this check `v`?
```suggestion
            if v.is_a?(Gem::Version)
```